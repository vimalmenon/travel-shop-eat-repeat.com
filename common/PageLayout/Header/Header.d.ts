import { IMetaData } from '@types';

export interface IHeaderProps {
  navigation: IMetaData[];
}
