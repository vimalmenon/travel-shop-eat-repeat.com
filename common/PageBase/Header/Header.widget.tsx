export const Header: React.FC = () => {
  return <header>This is Header</header>;
};
