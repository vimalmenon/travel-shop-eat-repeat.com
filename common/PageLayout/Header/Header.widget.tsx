export const Header: React.FC = () => {
  return <header className="flex text-sm py-1 px-2">This is Header</header>;
};
