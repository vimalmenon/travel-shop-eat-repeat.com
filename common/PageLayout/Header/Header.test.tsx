import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { Header } from './Header.widget';

describe('common > <PageBase> <Header> Component', () => {
  describe('Snapshot Testing', () => {
    it('<Header>', () => {
      const wrapper = shallow(<Header />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
