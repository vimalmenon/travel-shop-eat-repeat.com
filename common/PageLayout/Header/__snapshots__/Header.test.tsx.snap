// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common > <PageBase> <Header> Component Snapshot Testing <Header> 1`] = `
<header
  className="flex text-sm py-1 px-2"
>
  This is Header
</header>
`;
